rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -800,6 +801,37 @@ int LuaSyncedCtrl::SetAlly(lua_State* L)
 }
 
 
+/*** Changes the start box position of an allyTeam.
+ *
+ * @function Spring.SetAllyTeamStartBox
+ * @number allyTeamID
+ * @number xMin left start box boundary
+ * @number zMin top start box boundary
+ * @number xMax right start box boundary
+ * @number zMax bottom start box boundary
```

Comment:
"in elmos" (since this is different from startscript which wants 0-1)

Reply:
Ah so that's the term to use? 1 elmo = 1 square on the map, and thus the `x`/`z` positions are effectively elmos?

Reply:
yes https://beyond-all-reason.github.io/spring/articles/units-of-measurement

Reply:
Fixed!